test-suite-kotlin/src/test/kotlin/io/micronaut/kafka/docs/consumer/topics/ProductListener.kt

```diff
@@ -0,0 +1,25 @@
+package io.micronaut.kafka.docs.consumer.topics
+
+import io.micronaut.configuration.kafka.annotation.KafkaKey
+import io.micronaut.configuration.kafka.annotation.KafkaListener
+import io.micronaut.configuration.kafka.annotation.Topic
+import io.micronaut.context.annotation.Requires
+
+@Requires(property = "spec.name", value = "TopicsProductListenerTest")
+@KafkaListener
+class ProductListener {
+
+    // tag::multiTopics[]
+    @Topic("fun-products", "awesome-products")
+    fun receiveMultiTopics(@KafkaKey brand: String, name: String) {
+        println("Got Product - $name by $brand")
+    }
+    // end::multiTopics[]
+
+    // tag::patternTopics[]
+    @Topic(patterns = ["products-\\w+"])
+    fun receivePatternTopics(@KafkaKey brand: String, name: String) {
+        println("Got Product - $name by $brand")
```

Comment:
Us `LOG` not `println`

Reply:
changed it, here: https://github.com/micronaut-projects/micronaut-kafka/pull/832/commits/bfa2e2caf5233ff5064c5508d8b2eb2e08726b37